nodejs/extensions.bzl

```diff
@@ -62,6 +72,10 @@ If set then the version found in the .nvmrc file is used instead of the one spec
 This setting creates a dependency on a c++ toolchain.
 """,
             ),
+            "node_urls": attr.string_list(
+                doc = "Custom list of URL formats to use to download NodeJS, containing {version} and {filename}",
```

Comment:
Can you copy the same docs that are in the repo rule version?
```
List of URLs to use to download Node.js.

Each entry is a template for downloading a node distribution.

The `{version}` parameter is substituted with the `node_version` attribute,
and `{filename}` with the matching entry from the `node_repositories` attribute.
```

Reply:
Will do so tomorrow!

Reply:
Done!